utils/extensionRequests.js

```diff
@@ -16,12 +16,12 @@ const formatExtensionRequest = async (extensionRequest) => {
   if (!extensionRequest) return extensionRequest;
 
   let { timestamp, id, assignee, ...body } = extensionRequest;
-
+  const userId = assignee;
```

Comment:
Wouldn't `assigneeId` or `assigneeUserId` make more sense here?

Reply:
Thanks for the suggestion. I have made the change and it increases the readability of code now.